libcaf_net/caf/net/udp_datagram_socket.test.cpp

```diff
@@ -59,30 +68,30 @@ error read_from_socket(udp_datagram_socket sock, byte_buffer& buf) {
   return make_error(sec::runtime_error, "too many read attempts");
 }
 
-} // namespace
-
-CAF_TEST_FIXTURE_SCOPE(udp_datagram_socket_test, fixture)
+WITH_FIXTURE(fixture) {
 
-CAF_TEST(socket creation) {
+TEST("socket creation") {
   ip_endpoint ep;
-  CHECK_EQ(parse("0.0.0.0:0", ep), none);
-  CHECK(make_udp_datagram_socket(ep));
-}
+  check_eq(parse("0.0.0.0:0", ep), none);
+  check(static_cast<bool>(make_udp_datagram_socket(ep)));
+} // WITH_FIXTURE(fixture)
```

Comment:
I don't think this belongs here?

```suggestion
}
```